packages/cardano-services/docker-compose.yml

```diff
@@ -153,6 +155,34 @@ services:
         max-size: "200k"
         max-file: "10"
 
+  blockfrost-worker:
+    build:
+      context: ../../
+      target: blockfrost-worker
+    depends_on:
+      postgres:
+        condition: service_healthy
+    environment:
+      - LOGGER_MIN_SEVERITY=${LOGGER_MIN_SEVERITY:-info}
+      - BLOCKFROST_API_KEY=${BLOCKFROST_API_KEY}
```

Comment:
Right! I mainly thought to let it optional, I worked in following direction:
by default the docker compose should start normally; if enabled through environment, it can start with Blockfrost enabled

Reply:
By "secret" I mean, a [Docker Secret](https://docs.docker.com/engine/reference/commandline/secret/): here: https://github.com/input-output-hk/cardano-js-sdk/pull/619/files/131595f8409b0473bab790932475b3eaaa704b1e#diff-a5f22c45d19dce606e9bae0a8cd3a11ce586442845b16c7e1a056ac7e028ea79R186docker